src/Config/ConfigFile.php

```diff
@@ -18,6 +18,7 @@ public static function loadInDirectory(string $dir): array
     {
         $configFiles = current(array_filter(
             [
+                $dir . '/../../../../.xulieta.xml',
```

Comment:
This looks like a pyramid-scheme 🤣

Reply:
![kép](https://user-images.githubusercontent.com/952007/111212578-0c3dca80-85d0-11eb-8af5-1704dc385cf7.png)